frontend/public/src/components/EnrolledItemCard.js

```diff
@@ -88,6 +89,9 @@ export class EnrolledItemCard extends React.Component<
       if (isSuccessResponse(resp)) {
         messageType = ALERT_TYPE_SUCCESS
         userMessage = `You have been successfully unenrolled from ${enrollment.run.title}.`
+        if (typeof closeDrawer === "function") {
```

Comment:
One change for something that wasn't in the initial issue (but is related): can you apply the same change to the email settings notification? Should be largely this same code, but in the `onSubmit` function.

Reply:
@jkachel I've applied the same change to the email settings notification. I also moved this code outside if isSuccessResponse as I realized that the drawer should be closed even if there is an error so user can see the message.
Please take a look. thanks